.babelrc

```diff
@@ -1,3 +1,10 @@
 {
-  "presets": ["env"]
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
```

Comment:
is there a specific reason for this? My feeling is that if the user wants to use modules in their code, then they should think about how (if at all) they want to compile them. I don;t want to pre-empt that choice

Reply:
Doing this allows the `@babel/preset-env` preset to transpile to the JS target supported by the desired browser/node environment, _without_ first transpiling libraries & code for the build environment (recent Node.js versions handle ES6 imports/exports).
We can remove this if you think it's better to leave this up to the user.

Reply:
Actually Node doesn't fully support ES6 imports/exports. We can remove this setting for general consistency.